tests/gui/base/test_inputs.py

```diff
@@ -1,14 +1,16 @@
-from PyQt5.QtWidgets import QApplication, QLineEdit
+from PyQt5.QtWidgets import QApplication, QCheckBox, QLineEdit
 
 from securedrop_client.gui.base import PasswordEdit
 
 app = QApplication([])
 
 
 def test_PasswordEdit(mocker):
+    checkbox = QCheckBox()
     passwordline = PasswordEdit(None)
-    passwordline.togglepasswordAction.trigger()
 
+    passwordline.on_toggle_password_Action()
+    checkbox.isChecked()
```

Comment:
Does `checkbox.isChecked()` do anything in this context? Was it meant to be an assertion, or maybe a debug statement?